src/main/java/net/mcreator/workspace/WorkspaceFileManager.java

```diff
@@ -72,7 +72,7 @@ public class WorkspaceFileManager implements Closeable {
 
 		// start autosave scheduler
 		lastSchedule = dataSaveExecutor.schedule(new SaveTask(this),
-				PreferencesManager.PREFERENCES.backups.workspaceAutosaveInterval, TimeUnit.SECONDS);
+				PreferencesManager.PREFERENCES.backups.workspaceAutosaveInterval.get().longValue(), TimeUnit.SECONDS);

```

Comment:
Those calls to long or int value ar not really needed (unnecessary unboxing). Probably in some other files too